src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/DesignTimeInputsChangeTracker.cs

```diff
@@ -146,19 +148,32 @@ internal void ProcessFileChangeNotification(IProjectVersionedValue<string[]> arg
         }
 
         internal void ProcessDataflowChanges(IProjectVersionedValue<ValueTuple<DesignTimeInputs, IProjectSubscriptionUpdate>> input)
+        {
+            _currentState = GenerateOutputData(_currentState, input);
+
+            // Processed bad input data
+            if (_currentState is null)
+            {
+                // Lets return a default value to avoid hangs waiting on this.
+                _currentState = new DesignTimeInputsDelta(EmptyCollections.OrdinalStringSet,
+                    EmptyCollections.OrdinalStringSet, Enumerable.Empty<DesignTimeInputFileChange>(), string.Empty);
```

Comment:
Are we sure that downstream consumers of this data are able to handle empty state?

Reply:
Not sure If BuildDesignTimeOutput() should return "" when AppliedValue.Value.TempPEOutputPath. 

`BuildDesignTimeOutput()` -> `GetDesignTimeInputXmlAsync()` -> `GetDesignTimeInputXmlAsync()` -> [GetOutputFileNameFromRelativePath](https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/DesignTimeInputsCompiler.cs#L277)( "" ) => return Path.Combine(**tempPEOutputPath**, relativePath.Replace('\\', '.') + ".dll");